control_panel_api/tests/test_views.py

```diff
@@ -11,24 +11,28 @@
 )
 from rest_framework.test import APITestCase
 
+from .aws_mock import AWSClientPatcher, aws
 from control_panel_api.models import (
     AppS3Bucket,
     S3Bucket,
     User,
     UserApp,
     UserS3Bucket,
 )
+from .helm_mock import HelmClientPatcher, helm
```

Comment:
The naming is now confusing as to what is real or mocked.

Reply:
Yep, I'll rename the module

Reply:
👍

Reply:
I've removed them completely and refactored to use class decorators.